src/scenarios/shared/postcommands.py

```diff
@@ -1,8 +1,31 @@
 from shared.const import APPDIR, TMPDIR, TRACEDIR, PUBDIR, BINDIR, CROSSGENDIR
 from performance.common import remove_directory
+from argparse import ArgumentParser
+import subprocess
+
+class PostCommands:
+    '''
+    Handles any post run cleanup
+    '''
+
+    def __init__(self):
+        parser = ArgumentParser()
+
+        parser.add_argument('--readonly-dotnet',
+                            dest='readonly_dotnet',
+                            default=False,
```

Comment:
nit: This is not strictly needed as the default of the `store_true` action is false.